internal/log_unix.go

```diff
@@ -29,6 +31,17 @@ import (
 // so we just exit with the error
 func SetupHookLogging(hooks []config.LogrusHook, componentName string) {
 	logrus.SetReportCaller(true)
+	logrus.SetFormatter(&utcFormatter{
```

Comment:
If you move this back to `internal/log.go` and undo the changes to `setup/base/base.go`, I'm happy with this. :)